internal/web/results.go

```diff
@@ -150,6 +150,49 @@ func Results(w http.ResponseWriter, r *http.Request) {
 	return
 }
 
+func notValidatedYet(w http.ResponseWriter, r *http.Request, badge []byte, validator string, user, repo string) {
```

Comment:
You can simplify the signature further by combining the `validator`, `user` and `repo` arguments as `string`: `... []byte, validator, user, repo string) {`.